conmon/conmon.c

```diff
@@ -545,6 +623,33 @@ int main(int argc, char *argv[])
 		}
 	}
 
+	/* Setup OOM notification for container process */
+	memory_cgroup_path = process_cgroup_subsystem_path(cpid, "memory");
+	if (!memory_cgroup_path) {
+		nexit("Failed to get memory cgroup path");
+	}
+
+	bool oom_handling_enabled = true;
+	char memory_cgroup_file_path[PATH_MAX];
+	snprintf(memory_cgroup_file_path, PATH_MAX, "%s/cgroup.event_control", memory_cgroup_path);
+	if ((cfd = open(memory_cgroup_file_path, O_WRONLY)) == -1) {
+		nwarn("open cgroup.event_control");
```

Comment:
Fixing the message. But according to man page -1 is fine. See http://man7.org/linux/man-pages/man2/open.2.html